src/pages/ProductsPage/ProductsPageContainer.tsx

```diff
@@ -1,14 +1,16 @@
-import PageOne from "./PageOne"
+
 import { useAppDispatch, useAppSelector } from "../../app/hooks"
 import { RootState } from "../../app/store";
 import { getProducts } from "../../features/products/CatalogApiService";
+import ProductsPage from "./ProductsPage";
 
 
-export const PageOneContainer = () => {
+export const ProductsPageContainer = () => {
     const dispatch = useAppDispatch();
     const products = useAppSelector((state: RootState) => state.catalog.products);
```

Comment:
Map the valid object for component here.
Example:
`const products: ProductModel[] = useAppSelector((store) => store.catalog.productsIds.map((id) => store.catalog.products[id]) );`

Reply:
Props interface has been changed, prop "products" has "ProductModel []" type.
New mapping: const products: _**ProductModel[] = useAppSelector((state: RootState) =>
    state.catalog.productsIds.map((id) => state.catalog.products[id])
  );**_